deps.edn

```diff
@@ -597,7 +597,7 @@
   :profile
   {:main-opts ["-m" "metabase.core" "profile"]
    :jvm-opts  ["-XX:+CITime" ; print time spent in JIT compiler
-               "-Xlog:gc"]}
```

Comment:
this also worth keeping because log:gc is deprecated

Reply:
https://github.com/metabase/metabase/pull/31732/commits/b37629c017bd81b1696a2c784123b1765d8c753c